CHANGELOG.md

```diff
@@ -9,6 +9,24 @@ NEW FEATURES:
 
     Unlike the existing `precondition` and `postcondition` blocks, Terraform will not halt execution should the scoped data block fail or error or if any of the assertions fail. 
     This allows practitioners to continually validate the state of their infrastructure outside the usual lifecycle management cycle. 
+    
+* `import` blocks for importing infrastructure: Root module authors can now use the `import` block to declare their intent that Terraform adopt an existing resource. 
```

Comment:
s/`import`/`import{}`?

> to declare their intent that Terraform adopt an existing resource. 

-> to declare their intent that Terraform manage an existing, unmanaged resource.

Reply:
I can't think of any other place where we've referred to a block type by writing a pair of braces immediately after it. If we're going to adopt that convention we should probably do so holistically.

Reply:
Ah, okay, ignore then!